clientbuffer.cpp

```diff
@@ -52,9 +52,14 @@ class CClientBufferMod : public CModule
     MODCONSTRUCTOR(CClientBufferMod)
     {
         AddHelpCommand();
-        AddCommand("AddClient", static_cast<CModCommand::ModCmdFunc>(&CClientBufferMod::OnAddClientCommand), "<identifier>", "Add a client.");
-        AddCommand("DelClient", static_cast<CModCommand::ModCmdFunc>(&CClientBufferMod::OnDelClientCommand), "<identifier>", "Delete a client.");
-        AddCommand("ListClients", static_cast<CModCommand::ModCmdFunc>(&CClientBufferMod::OnListClientsCommand), "", "List known clients.");
+        AddCommand("AddClient", t_d("<identifier> [timelimit]"), t_d("Add a client with optiona time limit."),
+                   [=](const CString& sLine) { OnAddClientCommand(sLine); });
+        AddCommand("DelClient", t_d("<identifier>"), t_d("Delete a client."),
+                   [=](const CString& sLine) { OnDelClientCommand(sLine); });
+        AddCommand("ChangeClient", t_d("<identifier> [timelimit]"), t_d("Change a clients time limit."),
```

Comment:
If more per-client parameters are added, they would all have to be specified all at once in the `ChangeClient` command, including the old values which the user doesn't want to change. I think it would be better if there was a `SetClientTimeLimit` command instead.

Reply:
Yes, I have been considering that and you are right here. But it's just a command name change or you would see a different logic here?

Reply:
Yes, in this case, it would be just a command name change.